axum-extra/src/extract/multipart.rs

```diff
@@ -398,4 +399,36 @@ mod tests {
         let res = client.post("/").multipart(form).await;
         assert_eq!(res.status(), StatusCode::PAYLOAD_TOO_LARGE);
     }
+
+    #[tokio::test]
+    async fn body_too_large_with_tracing() {
+        tracing::subscriber::set_global_default(
+            tracing_subscriber::FmtSubscriber::builder()
```

Comment:
You might want to use something like [this](https://docs.rs/tracing/latest/tracing/subscriber/fn.with_default.html) instead. If someone added a second test like this, the initialization would panic.

Also, I don't think we need `tracing` dev dependency just `tracing-subscriber`.

Reply:
If the `tracing` dev-dep is removed, this test needs to have `#[cfg(feature = "tracing")]`. But that probably makes sense anyways.